packages/browser/src/login/oidc/redirectHandler/AuthCodeRedirectHandler.ts

```diff
@@ -69,6 +69,48 @@ export async function exchangeDpopToken(
   });
 }
 
+// We
+const DEFAULT_LIFESPAN = 1800;
+
+/**
+ * Stores the resoure server session information in local storage, so that they
+ * can be leveraged on refresh.
+ * @param webId
+ * @param authenticatedFetch
+ * @param storageUtility
+ */
+async function setupResourceServerSession(
+  webId: string,
+  authenticatedFetch: typeof fetch,
+  storageUtility: IStorageUtility
+): Promise<void> {
+  const webIdAsUrl = new URL(webId);
+  const resourceServerIri = webIdAsUrl.origin;
+  const resourceServerResponse = await authenticatedFetch(
+    `${resourceServerIri}/session`
+  );
+  if (
+    resourceServerResponse.status === 404 ||
```

Comment:
These checks seem overly-specific - i.e., is the condition really **_anything but_** a 200? For example, if the server returned a 500, or a 303 or something, we'd be assuming all was well, right...?

Reply:
That's a good point. I fixed this in 93fb34aa9bff1c435d9d3f2ea46e08bb64b3b420